btr-web/btr-main-app/lang/en.json

```diff
@@ -33,7 +33,8 @@
         "sentence2": "Indicating that it is not possible to obtain or confirm information about an individual constitutes a statement in respect of a material fact."
       }
     },
-    "unknown": "Unknown"
+    "unknown": "Unknown",
+    "certifySection": "2. Certify"
```

Comment:
order number should not be part of the translation. It might be 3rd section, 5th, etc... but "certify" will still be translated the same.